data/India/Lok_Sabha/ep-popolo-v1.0.json

```diff
@@ -72167,6 +72167,19 @@
     },
     {
       "classification": "general election",
+      "end_date": "1999",
+      "id": "Q17056099",
+      "identifiers": [
+        {
+          "identifier": "Q17056099",
+          "scheme": "wikidata"
+        }
+      ],
+      "name": "Indian general election in Andhra Pradesh, 1999",
```

Comment:
This is just part of a General Election, not an actual instance of one. I've corrected that upstream.